source/03-specifications/02-formats-for-counter-reports.rst

```diff
@@ -116,8 +116,12 @@ Table 3.f (below): COUNTER Report Header Elements
      - EBSCO Information Services\ |br|\ |lb|
        360 COUNTER
 
+   * - Registry
+     - The link to the platform's COUNTER Registry record.
```

Comment:
Maybe it should be clarified that COUNTER compliant content providers MUST include the link, and that content provider that don't have a Registry record MUST NOT include a value in their reports?